src/classes.js

```diff
@@ -2,4 +2,62 @@ class Foo {
 
 }
 
-var bar;
+var bar = new Foo();
+
+class Dog {
+  constructor(says) {
+    this.says = 'life is ruff'
+  }
+}
+var dog = new Dog;
+
+class Cat {
+
+  constructor(garfield) {
+    this.garfield = garfield
+    this.growl = function(growl) {
+      return 'meow'
+    }
+  }
+}
+
+var garfield = new Cat();
+
+class FacelessMan {
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+var aMan = new FacelessMan(`Jaqen H'ghar`);
+
+class Lion extends Cat {
+  constructor() {
+    super();
+    this.size = 'BIG';
+  }
+}
+
+
+class KeepSecret {
+  constructor() {
+    this.squeal = function() {
+      return `My class rocks!`
```

Comment:
Don't return a pure string. Instead this should return the secret that was passed in when the `KeepSecret` object was created.